docker-compose.yml

```diff
@@ -27,3 +27,15 @@ services:
     volumes:
       - $ROOT/demo:/orderly
       - $ROOT/config/orderlyweb:/etc/orderly/web
+  mq:
+    image: redis
+    ports:
+      - "6379:6379"
+  flower:
+    image: mher/flower:0.9.5
+    ports:
+      - "5555:5555"
```

Comment:
As above, though may be worth exposing to host via `127.0.0.1:5555:5555` as Flower appears to have a web interface